extensions/amp-story/1.0/media-tasks.js

```diff
@@ -109,200 +108,132 @@ function copyAttributes(fromEl, toEl) {
 }
 
 /**
- * Base class for tasks executed in order on HTMLMediaElements.
+ * A task executed in order on HTMLMediaElements.
+ * @interface
  */
 export class MediaTask {
-  /**
-   * @param {string} name
-   * @param {!Object=} options
-   */
-  constructor(name, options = {}) {
```

Comment:
`options` is unnecessary since only one task uses it. Other tasks take parameters directly, which minifies better. This task was refactored to match.